src/main/java/org/opentripplanner/routing/core/Fare.java

```diff
@@ -15,7 +15,7 @@
 public class Fare {
 
     public static enum FareType implements Serializable {
-        regular, student, senior, tram, special, youth
+        regular, student, senior, tram, special, youth, orcaRegular, orcaYouth, orcaLift, orcaSenior
```

Comment:
Instead of saying `orca*`, these "integrated" or "coordinated" fares could be named `integratedRegular` or `coordinatedRegular`, etc. or something like that to distinguish them from regular cash prices, that we can reuse with other agencies.

Reply:
I still think the `orca` prefix should be changed so we can write preferred fares if people are using electronic payment... so two other prefix suggestions in addition to the ones above: `preferred` (`preferredRegular`,...), `electronic` (`electronicLift`, ...)

Reply:
Gone with `electronicRegular`, `electronicYouth`, `electronicSpecial` and `electronicSenior`